app/models/queue_item.rb

```diff
@@ -0,0 +1,16 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+  
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+  
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
Another way to do this is with the new `find_by` method: `review = Review.find_by(user_id: user.id, video_id: video.id)`.